src/app/components/client/FixNavigation.tsx

```diff
@@ -64,49 +69,79 @@ export const Steps = (props: {
     return value.length > 0;
   }).length;
 
+  const completedScan = hasCompletedStep(props.data, "Scan");
+  const completedHighRisk =
+    hasCompletedStep(props.data, "HighRiskSsn") &&
+    hasCompletedStep(props.data, "HighRiskCreditCard") &&
+    hasCompletedStep(props.data, "HighRiskBankAccount") &&
+    hasCompletedStep(props.data, "HighRiskPin");
```

Comment:
Nit: What do you think about adding a helper function (that we could call `hasCompletedSteps`) that takes an array instead of chaining together the `hasCompletedStep` checks?

Reply:
An array seems too generic to me, but how about a helper function for the top-level sections? Added in c4678a818.

Reply:
Nice, I like it!